### PR TITLE
Refactors code to remove unneeded class and tidy session creation

### PR DESCRIPTION
**Pull request by jwgwalton**

### CODE REVIEW

File: application/views/template_view.py [hunk 1]
```diff
@@ -1,19 +1,19 @@
...
 template_view = Blueprint('template_view', __name__)
 
 
 @template_view.route('/template/<template_id>', methods=['POST'])
 def upload_template(template_id):
     template = request.get_json()
-    msg = TemplateController.upload_comms_template(template_id, template)
-    return make_response(msg, 201)
+    template_controller.upload_comms_template(template_id, template)
+    return make_response('', 201)
```
Comment (jwgwalton) [inline application/views/template_view.py]: @benjefferies make_response requires an argument here even if its an empty string.

File: application/controllers/template_controller.py [hunk 2]
```diff
@@ -2,22 +2,21 @@
...
 
 
 logger = get_logger()
 
-UPLOAD_SUCCESSFUL = 'The upload was successful'
 PREEXISTING_TEMPLATE = 'ID already exists'
 
 
-def get_template_by_id(template_id, session):
+def get_template_by_id_from_db(template_id):
```
Comment (benjefferies) [inline application/controllers/template_controller.py]: Don't think we need the from_db in the method name

**Review by benjefferies — APPROVED** [review]
Great stuff. Minor naming issue

**Review by rhiGriff — CHANGES_REQUESTED** [review]
Can we update the logging standards to the SDC logging standards documented here: https://digitaleq.atlassian.net/wiki/spaces/SDC/pages/69632229/Logging ?

File: application/controllers/template_controller.py (hunk 2 above)
Comment (jwgwalton) [reply application/controllers/template_controller.py]: I wanted to be explicit compared to the get_comms_template_by_id

**Comment by jwgwalton** [comment]
Hi @rhiGriff I can't actually see those logging standards.

File: application/controllers/template_controller.py (hunk 2 above)
Comment (benjefferies) [reply application/controllers/template_controller.py]: Fair enough I don't think it's detail that needs to be added. I guess you could say would putting API in the method name of all of the flask routes

**Comment by benjefferies** [comment]
Good spot @rhiGriff. It's also documented on a branch here https://github.com/ONSdigital/software-engineer-community/blob/add-logging-example/standards/logging.md

**Pull request merged**